bindings/python/tests/model/test-py-geos.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 - 2024 Geode-solutions
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+import sys
+import platform
+if sys.version_info >= (3, 8, 0) and platform.system() == "Windows":
+    for path in [x.strip() for x in os.environ['PATH'].split(';') if x]:
+        os.add_dll_directory(path)
+
+import opengeode
+import opengeode_io
+import opengeode_geosciences as geosciences
+import opengeode_geosciencesio_py_model as geosciences_io
+
+if __name__ == '__main__':
+    geosciences_io.GeosciencesIOModelLibrary.initialize()
+    test_dir = os.path.dirname(__file__)
+    data_dir = os.path.abspath(os.path.join(
+        test_dir, "../../../../tests/data"))
+
+    model = geosciences.load_structural_model(
+        os.path.join(data_dir, "test.lso"))
+    exporter = geosciences_io.BRepGeosExporter(model,"brep_to_geos")
+    #exporter.run()
```

Comment:
Why the run() are commented?

Reply:
because it does not work on stratigraphic model. I need to further investigate.